### PR TITLE
add simple makefile

### LINKED ISSUE #122: Create database docker on tests startup
At the moment I use test database that is created together with other services and runs all the time. Change it so that database is started on test startup and closed after test are finished.

Useful links:
https://divan.github.io/posts/integration_testing/

### LINKED ISSUE #148: Add makefile for the project

### PR DESCRIPTION
**Pull request by Prytu**
fixes #148 
fixes #122

### CODE REVIEW

File: backend/Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+install:
+	go install
+
+test:
+	docker-compose -f ../docker/test.yml build && docker-compose -f ../docker/test.yml up -d
```
Comment (VirrageS) [inline backend/Makefile]: You have to make sleep after this command since postgres has few second init time

File: backend/Makefile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+install:
+	go install
```
Comment (VirrageS) [inline backend/Makefile]: It would be convient if install also would fire `go get`

**Comment by Prytu** [comment]
So I think this would be the super simple initial verson of makefile.
Do you think I should move the testing part: starting docker, waiting for postgres startup, running tests and stopping docker script to another file and call the script from makefile? Or it is OK to keep it in makefile?

**Comment by VirrageS** [comment]
No, I think it is good idea to keep everything in makefile, it makes project structure cleaner.

**Pull request merged**